sdk/servicebus/ci.yml

```diff
@@ -39,3 +39,12 @@ extends:
       - name: azure-messaging-servicebus
         groupId: com.azure
         safeName: azuremessagingservicebus
+    AdditionalModules:
+        - name: perf-test-core
+          groupId: com.azure
+        - name: azure-messaging-servicebus-track1-perf
+          groupId: com.azure
+        - name: azure-messaging-servicebus-track2-perf
+          groupId: com.azure
+        - name: azure-storage-perf
```

Comment:
Do we need it here?

Reply:
This was an issue sometime back but Jim's PR fixed it. So I can remove it now . https://github.com/Azure/azure-sdk-for-java/pull/14035